app/Http/Requests/User/ProfileRequest.php

```diff
@@ -66,17 +62,23 @@ public function rules()
             ];
         }
 
-        if ($this->segment(1) == 'auth') {
+        if ($this->segment(1) == 'auth' && $value == '0') {
```

Comment:
if $value is boolean no need to do == it will be already considered as true or false

Reply:
Can you explain?